compiler/backend/closLangScript.sml

```diff
@@ -5,6 +5,8 @@
 *)
 open preamble backend_commonTheory;
 
+local open astTheory in end
```

Comment:
Surely it should be backed_common that becomes the grammar ancestor.  Why is `ast` chosen here?

Reply:
It's not a grammar ancestor in the sense of `set_grammar_ancestry`... But as for why I'm creating a dependency here, maybe see what happens if you remove it and try to build the file? (I'm not sure what will happen but I would guess the build fails.)